website/docs/jellyfish/api/loan.md

```diff
@@ -269,12 +269,55 @@ Creates a vault transaction.
 interface loan {
   createVault (vault: CreateVault, utxos: UTXO[] = []): Promise<string>
 }
+
 interface CreateVault {
   ownerAddress: string
   loanSchemeId?: string
 }
+
+interface UTXO {
+  txid: string
+  vout: number
+}
+```
+
+## depositToVault
+
+Deposit to vault.
+
+```ts title="client.loan.depositToVault()"
+interface loan {
+  depositToVault (vault: DepositVault, utxos: UTXO[] = []): Promise<string>
+}
+
+interface DepositVault {
+  id: string
+  from: string
+  amount: string // amount@symbol
+}
+
 interface UTXO {
   txid: string
   vout: number
 }
-```
\ No newline at end of file
+```
+
+## takeLoan
+
+Take loan.
+
+```ts title="client.loan.takeLoan()"
+interface loan {
+  takeLoan (metadata: TakeLoanMetadata, utxos: UTXO[] = []): Promise<string>
+}
+
+interface TakeLoanMetadata {
+  vaultId: string
+  amounts: string // amount@symbol
```

Comment:
I am thinking why `amounts` here is not `amount` like https://github.com/DeFiCh/jellyfish/pull/661/files#diff-3e4e57ab619608405a6f96e0ac91e233a158743d1f3ed54e16cdadffec038e08R296

I shall ask C++ programmers on this

Reply:
i think that is typo

Reply:
For your information, yesterday they told me that It takes array as well. it's a string that takes array format.
But, I don't think so.